kbase-extension/static/kbase/js/widgets/appWidgets2/input/toggleButtonInput.js

```diff
@@ -1,26 +1,25 @@
-/*global define*/
 /*jslint white:true,browser:true*/
```

Comment:
in fact, a global search/replace for `^\s*/* ?jslint.*?$` might be a good idea

Reply:
Next PR!